component/vdr/orb/vdr.go

```diff
@@ -417,9 +429,27 @@ func (v *VDR) Read(did string, opts ...vdrapi.DIDMethodOption) (*docdid.DocResol
 		return nil, fmt.Errorf("failed to fetch correct did from min resolvers")
 	}
 
+	if err := v.checkUnanchoredDIDTime(docResolution.DIDDocument); err != nil {
+		return nil, err
+	}
+
 	return docResolution, nil
 }
 
+func (v *VDR) checkUnanchoredDIDTime(didDocument *docdid.Doc) error {
+	if (strings.HasPrefix(didDocument.ID, fmt.Sprintf("did:%s:%s", DIDMethod, httpsProtocol)) ||
+		strings.HasPrefix(didDocument.ID, fmt.Sprintf("did:%s:%s", DIDMethod, "interim"))) &&
```

Comment:
We have been using "uAAA" instead of "interim" for months now. How old is this version?

Reply:
it's typo